style.css

```diff
@@ -0,0 +1,759 @@
+  }
+
+  .s-nav {
+    display: block;
+  }
+
+  .cards {
+    background-color: #27374d;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    padding: 5px;
+    margin: 5px;
+  }
+
+  .main-container {
+    background-color: #27374d;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    text-align: center;
+    padding: 5px;
+    margin: 5px;
+    border: 2px solid orangered;
+  }
+
+  .images {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 120px;
+  }
+
+  .plates {
+    display: flex;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+    width: 100%;
+    gap: 2rem;
+  }
+
+  .master-container {
+    width: 45%;
+  }
+
+  .none {
+    display: block;
+  }
+
+  .more {
+    margin-left: 50px;
+  }
+
+  .first-line {
+    display: none;
+  }
+
+  .message {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    /* width: 50%; */
+    text-align: center;
+    margin: 10px 100px;
+  }
+
+  .line-resto {
+    margin-bottom: 30px;
+  }
+
+  .pgraph {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    line-height: 1.5rem;
+    font-family: 'lato', sans-serif;
+    border: 3px solid white;
+    background-color: whitesmoke;
+    padding: 10px;
+    margin: 30px;
+  }
+
+  .first-section-paragraph {
+    margin-bottom: 0;
+  }
+
+  .votting-logo {
+    text-align: center;
+    margin: 10px 10px 40px 10px;
+  }
+
+  .logo-picture {
+    border: 2px solid grey;
+    width: 60%;
+    margin: auto;
+    padding: 10px;
+  }
+
+  .h-rule {
+    margin-top: 60px;
+  }
+
+  .burgers-pizz {
+    font-size: 20px;
+    margin: 10px 80px 40px 80px;
+  }
+
+  .double-picture {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 10px;
+    gap: 2rem;
+  }
+
+  .partner {
+    padding: 20px;
+    font-size: 30px;
+    color: white;
+  }
+
+  .footer {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    /* background-color: grey; */
+    gap: 4rem;
+    font-size: 25px;
+  }
+
+  .tof {
+    width: 15%;
+    margin-left: 80px;
+    padding: 20px;
+  }
+
+  .last-end {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: grey;
+    gap: 4rem;
+    font-size: 25px;
+  }
+}
+
+/* .paragraph {
+    display: none;
+  } */
+
+/* .button-more {
+  display: none;
+} */
+
+.last-end {
+  padding: 30px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #414246;
+  gap: 4rem;
+  font-size: 15px;
+}
+
+.last-section {
+  margin-left: 0;
+  background-color: #414246;
+  margin-top: 50px;
+  margin-bottom: 50px;
+}
+
+.l-img {
+  width: 15%;
+}
+
+.footers {
+  display: flex;
+  align-items: center;
+  gap: 4rem;
+  font-size: 25px;
+  margin-left: 50px;
+  margin-top: 50px;
+  margin-bottom: 50px;
+  justify-content: space-around;
+}
```

Comment:
- [ ]  Great work on your project, but according to your previous reviewer kindly add the background color to the entire footer. Additionally, it would be wonderful if you could double-check the color of the text, it's very important to stick to the design,
- kindly follow according to the project requirement The pages should look almost identical to the original design, outside the content differences.
The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design.

Basic color swatch: #272A31, #EC5242, #D3D3D3


The pages use the same fonts or similar fonts from the original design.

Fonts used originally: Cocogoose, Lato

- All page elements have similar spacing, borders, shadows, and paddings to the original design.
- [ ] your footer
![08 07 2023_23 29 52_REC](https://github.com/Young-Gaius/capstone-project/assets/92458236/b3901582-c530-4ad0-9c27-82c77490f03a)

- [ ] Behance template:
![Screenshot 2023-07-08 234128](https://github.com/Young-Gaius/capstone-project/assets/92458236/309b8f12-e1a5-4367-9ff5-25d44cea5a85)


- [ ] [ See Previous Review](https://github.com/Young-Gaius/capstone-project/pull/1#discussion_r1257377604)